src/app/common/stores/applications/Distribution/Distribution.store.ts

```diff
@@ -68,7 +68,7 @@ class DistributionStore extends BaseStore {
       const contract = this.distributionContract;
       const walletStore: WalletStore = this.getStore('Wallet');
       const result = await contract.methods.cumulativeClaimed(this.userAddress).call();
-      this.claimedRewards = walletStore.fromWei(parseInt(String(result)).toString());
+      this.claimedRewards = Number(walletStore.fromWei(parseInt(String(result)).toString()));
```

Comment:
I think there is some redundancy here. Turning the result into string, then int, then back to string.

Reply:
I'll create ticket for this to not waste a time now